flashinfer/comm/trtllm_alltoall.py

```diff
@@ -296,13 +296,15 @@ class MnnvlMoe:
     moe_mapping: Mapping = None
 
     @staticmethod
-    def get_moe_workspaces(mapping: Mapping):
+    def get_moe_workspaces(mapping: Mapping, config: Optional[MnnvlConfig]=None):
```

Comment:
Thanks for fixing the MPI import. The other improvements I can think of:
1. Allow users to just pass a torch distributed group. We can provide a `TorchDistributedBackend` implementation of `CommBackend`, and automatically wrap the torch group with it, if provided.
2. Eliminate `Mapping` arg by creating it internally from the user provided distributed group, using group.world_size, group.rank, torch.cuda.device_count().